src/app/code/community/Zendesk/Zendesk/Model/Observer.php

```diff
@@ -157,20 +157,24 @@ public function changeIdentity(Varien_Event_Observer $event)
         
         if (is_object($order_totals)) {
             $order_totals
-            ->addFieldToSelect('*')
-            ->addFieldToFilter('customer_id', $customer->getId())
-            ->addFieldToFilter('status', Mage_Sales_Model_Order::STATE_COMPLETE)
-            ->addAttributeToSelect('grand_total')
-            ->getColumnValues('grand_total');
+                ->addFieldToFilter('customer_id', $customer->getId())
+                ->addFieldToFilter('status', Mage_Sales_Model_Order::STATE_COMPLETE)
+            ;
```

Comment:
Semi-colon needs to be aligned. 

Can we do this minor change while bumping versions? @jwswj